packages/tx/src/legacyTransaction.ts

```diff
@@ -367,6 +361,15 @@ export default class Transaction extends BaseTransaction<Transaction> {
           `Legacy txs need either v = 27/28 or v >= 37 (EIP-155 replay protection), got v = ${v}`
         )
       }
+      if (
+        common?.hardfork() === 'chainstart' ||
+        common?.hardfork() === 'homestead' ||
+        common?.hardfork() === 'tangerineWhistle'
+      ) {
+        if (v > 28) {
+          throw new Error('Legacy txs need either v = 27/28 or v >= 37 (EIP-155 replay protection)')
+        }
+      }
```

Comment:
This is both not correct (legacy txs can still be used on other hardforks) and not reachable, since all v cases caught here are already included in the conditional clause from above.

So this can be completely removed. 🙂

Reply:
Without this, 90 TX tests fail like this for Homestead, Frontier, and EIP-150. 
```
not ok 2937 Transaction should be invalid on Homestead
  ---
    operator: ok
    expected: true
    actual:   false
```
So either the tests themselves are wrong, or something is not being caught by the rest of the conditional for another reason

Reply:
The way that the TX test work, using the `transactionRunner`, I'm finding it very hard to track down exactly which test is which